error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/AssertJMapRulesTestInput.java

```diff
@@ -131,7 +131,7 @@ MapAssert<Integer, Integer> testAbstractMapAssertHasSameSizeAs() {
   }
 
   AbstractAssert<?, ?> testAssertThatMapContainsOnlyKeys() {
-    return assertThat(ImmutableMap.of(1, 2).keySet()).hasSameElementsAs(ImmutableSet.of(1));
+    return assertThat(ImmutableMap.of(1, 2).keySet()).hasSameElementsAs(ImmutableSet.of(3));
```

Comment:
Thanks 🚀 
I have seen this pattern everywhere but I cant fully understand the reasning behind this.
Why arent we using correct expected values ? 🤔

Reply:
Good question. I wouldn't say we're fully consistent in this respect. But in new code I try to use distinct values, so that we properly validate that Refaster correctly "shuffles things around". (And since the code isn't executed anyway... :upside_down_face:.)